backend/src/types.d.ts

```diff
@@ -16,4 +16,10 @@ export type meta = {
   dateEdited: string
   version: string
   automatariumVersion: string
+}
+
+declare module 'express-serve-static-core' {
+  interface Request {
+    uid?: string
+  }
```

Comment:
I've genuinely not known how to do this in the past XD - makes sense in retrospect